src/styles/mixins.less

```diff
@@ -157,6 +157,13 @@
   text-overflow: ellipsis;
 }
 
+.m-line-clamp(@count) {
+  display: -webkit-box;
+  -webkit-line-clamp: @count;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+}
```

Comment:
I didn't know all browser supported this. I was already wondering how you could do ellipsis on multiple lines. So cool :partying_face:

Reply:
Yes, I've discovered it some time last year, very useful for responsive UI especially 😎